Project4/style.css

```
@@ -0,0 +1,637 @@
+body {
+    background-color: black;
+    color: white;
+    font-family: 'Gugi', cursive;
```

While Google Fonts recommends "cursive" here, your font is more of a sans-serif than cursive, so I recommend updating the value here:
```suggestion
    font-family: 'Gugi', sans-serif;
```